include/ccl_cls.h

```diff
@@ -33,11 +35,11 @@ double ccl_angular_cl_limber(ccl_cosmology *cosmo,
  * For specific cases see documentation for ccl_error.c
  */
 void ccl_angular_cls_nonlimber(ccl_cosmology *cosmo,
-			       ccl_cl_tracer_collection_t *trc1,
-			       ccl_cl_tracer_collection_t *trc2,
-			       ccl_f2d_t *psp,
-			       int nl_out,int *l_out,double *cl_out,
-			       int *status);
+         ccl_cl_tracer_collection_t *trc1,
+         ccl_cl_tracer_collection_t *trc2,
+         ccl_f2d_t *psp,
+         int nl_out,int *l_out,double *cl_out,
```

Comment:
Add spaces to the arguments.

Reply:
bug @damonge about this ;P

Reply:
I've fixed as much as I can but I did not get everything.

Reply:
Sounds good. I spotted a bunch of other potential formatting improvements, but figured you didn't want to hear about them.

Reply:
Right. I've been looking for a good C linter but have not found one.

Reply:
I do couldn't care less. I personally prefer no spaces and consistent indentation (the latter is actually quite nightmarish right now).